js/gladiator.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+
+export class Gladiator{
```

Comment:
be sure to use export default class Gladiator{
to properly export the class if there is only one class in the file